docs/source/get-started/quickstart.md

```diff
@@ -151,7 +151,7 @@ user@host:~/lumigator$ curl -s http://localhost:8000/api/v1/jobs/evaluate/ \
 :::{tab-item} Python SDK
 :sync: tab2
 ```python
-from lumigator_schemas.jobs import JobType, JobEvalCreate
+from lumigator_schemas.jobs import JobType, JobEvalLiteCreate
```

Comment:
This file should describe the flow of an inference plus an evaluation (like in the CI test `test_upload_data_launch_job`). The old evaluation performed both tasks.

Reply:
Good point. Let me split this out into a separate PR so that it's easier to review that part and make sure we completely fix the docs. I think this part can come after the other PRs that fix the endpoint names etc. 

Created https://github.com/mozilla-ai/lumigator/issues/905